src/constants/commonOptions.js

```diff
@@ -19,6 +19,11 @@ module.exports = {
     placeholder: 'Placeholder',
     tooltip: 'The placeholder text that will appear when this field is empty.'
   },
+  description: {
```

Comment:
I added an extra bit of HTML to display the description when rendering a component, it's in my other pull request on the ngFormio repo. https://github.com/formio/ngFormio/pull/280/commits/321d9a36b675a0e8937eb25f6490c3ec8c917442#diff-cc84655aafd02adf5dd2a91991f14731